experimental/components/Cargo.toml

```diff
@@ -45,7 +45,7 @@ displaydoc = { version = "0.2.3", default-features = false }
 log = { version = "0.4", optional = true }
 num-bigint = "0.4.4"
 num-rational = "0.4.1"
-num-traits = "0.2.17"
+num-traits = "0.2.18"
```

Comment:
As discussed: please disable `default-features` on the num crates

Reply:
done